templates/common/config/00-config.conf

```diff
@@ -12,7 +12,7 @@ show_multiple_locations={{ .ShowMultipleLocations }}
 enabled_import_methods=[web-download,glance-direct]
 bind_host=localhost
 bind_port=9293
-workers=3
+workers={{ .Workers }}
```

Comment:
should we limit the max cap for workers same as the number of CPUs?

Reply:
we can yes, but we might need more logic in case we don't expose the parameter and allow customers to override it via customServiceConfig